ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py

```diff
@@ -1784,6 +1784,7 @@ def test_paddle_rsqrt(
         available_dtypes=helpers.get_dtypes("valid"),
     ),
 )
+
 def test_paddle_rsqrt_(
```

Comment:
Hi @ArsalanAli915, please resolve the lint errors found by the CI checks and then we can merge this. Thanks!

Reply:
Thanks for feedback @AnnaTz . I have ressolved the lint error. Lint error occurs for already presented ones.. pre-commit passes my proposed changes.. Also all test are passing. Here is SS of Lint pass.
![m2](https://github.com/unifyai/ivy/assets/103112521/fc88fd8a-e3c3-4f5e-85fa-83b31a6d0805)